lib/core/test/manual/Cardano/Launcher/POSIXSpec.md

```diff
@@ -21,7 +21,7 @@ signals are correctly handled on a unix system.
   ```
   $ ps -ef | grep cardano
   10983  4904  0 09:42 pts/1 00:00:00 /home/user/Documents/IOHK/cardano-wallet/.stack-work/install/x86_64-linux/lts-13.8/8.6.3/bin/cardano-wallet
-  11003 10983 75 09:42 pts/1 00:03:30 cardano-http-bridge start --port 8080
+  11003 10983 75 09:42 pts/1 00:03:30 cardano-wallet-jormungandr start --port 8080
```

Comment:
@Anviking we are sure here that we can replace `http-bridge` with `wallet-jormungandr` ?

Reply:
I replaced the commands in the guide carefully, verifying that they worked.

But my `ps -ef | grep cardano` output was a lot messier, so I didn't copy-paste, but rather just tweak the logs. The log-output-examples are therefore marginally incorrect.

c5cf30c